Code/JavaWrappers/gmwrapper/src-test/org/RDKit/MolStandardizeTest.java

```diff
@@ -26,7 +26,7 @@ public void testPipelineBadInput() {
 
 	    assertEquals(result.getStage(), PipelineStage.PARSING_INPUT);
         assertFalse(result.getStatus() == PipelineStatus.NO_EVENT);
-        assertTrue((result.getStatus() & PipelineStatus.INPUT_ERROR) != PipelineStatus.NO_EVENT);
+        assertTrue(PipelineStatus.swigToEnum(result.getStatus().swigValue() & PipelineStatus.INPUT_ERROR.swigValue()) != PipelineStatus.NO_EVENT);
```

Comment:
This feels like a lot of boilerplate. Is it really all required?

Reply:
It is equivalent to what you would need in C++ with an `enum class`:
```C++
// enum.cpp
#include <iostream>

#ifdef NO_STRONGLY_TYPED_CLASS
#define ENUM_TYPE enum
#else
#define ENUM_TYPE enum class
#endif

ENUM_TYPE MyEnum {
    value1 = 1 << 0,
    value2 = 1 << 1,
    value3 = value1 | value2
};

int main() {
#ifndef NO_ENUM_CAST
    auto res = (static_cast<MyEnum>((static_cast<int>(MyEnum::value1)
        | static_cast<int>(MyEnum::value2))) == MyEnum::value3);
#else
    auto res = (MyEnum::value1 | MyEnum::value2 == MyEnum::value3);
#endif
    std::cout << "res " << res << std::endl;
}
```
```
$ g++ -Wall enum.cpp -o enum
$ ./enum
res 1
$ g++ -DNO_ENUM_CAST -Wall enum.cpp -o enum
enum.cpp:14:32: error: no match for 'operator|' (operand types are 'MyEnum' and 'bool')
   14 |     auto res = (MyEnum::value1 | MyEnum::value2 == MyEnum::value3);
      |                 ~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                         |                       |
      |                         MyEnum           
$ g++ -DNO_ENUM_CAST -DNO_STRONGLY_TYPED_CLASS -Wall enum.cpp -o enum
$ ./enum
res 1
```